src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -1889,6 +1889,59 @@ impl EventProcessor {
     }
 }
 
+/// Recorded mouse delta designed to filter out noise.
+struct Delta<T> {
+    inner: Option<(T, T)>,
+}
```

Comment:
Put it somewhere else, we have entire directly with called `util`(the most useless name for directly btw) just laying around for stuff like that.

Reply:
Moved to `util/mouse.rs`